src/content/docs/concepts/the-graph.md

```diff
@@ -2,43 +2,23 @@
 title: The Graph
 ---
 
-Each program has a graph which is how you interact with it.
+Each Membrane program is its own _graph_, with data and functionality represented by nodes in the graph.
 
-A program's graph is defined by a schema defined in `memconfig.json`.
+<!-- TODO: visualization -->
 
-The schema is a set of types, each of which define fields, actions, and events.
+As a mental model, you can think of all your different Membrane programs as nodes in your overall Membrane graph, with each program node as its own subgraph. Your programs connect to one another—or even subnodes of another program—in this larger graph. For example, the `email` program that comes pre-installed might connect to many other programs that use it.
```

Comment:
@juancampa fyi — docs assumes that email, sms, drawer, http, and getting-started come pre-installed in a new user's Membrane workspace.